src/python/pants/util/contextutil.py

```diff
@@ -22,6 +22,13 @@
 from pants.util.tarutil import TarFile
 
 
+class InvalidZipPath(zipfile.BadZipfile):
```

Comment:
Although `open_zip` is `:API: public`, the API is not documented as raising anything in particular. I find no trapping of `BadZipfile` in our codebase so it is probably appropriate to switch `InvalidZipPath` to subclassing `ValueError` - which is more appropriate - and documenting the actual public api in `open_zip` if you're feeling charitable.